xmrstak/backend/amd/autoAdjust.hpp

```diff
@@ -52,14 +52,16 @@ class autoAdjust
 			return false;
 		}
 
-		devVec = getAMDDevices(0);
+		devVec = getAMDDevices(platformIndex);
 
 
 		int deviceCount = devVec.size();
 
 		if(deviceCount == 0)
+		{
+			printer::inst()->print_msg(L0,"WARNING: No AMD device found.");
             return false;
```

Comment:
Can you fix that indent?

Reply:
Done, after we added the most changes I will run `sed` over the code to remove all my spaces with taps.

Reply:
Thanks, keep in mind that is is much easier to go the other way round (sed s/\t/    /g), one of the reasons why I always use tabs. 

When changing spaces into tabs make sure to include ^ anchor to prevent random tabs in the code.